README.md

```diff
@@ -1,25 +1,70 @@
 # Marker
 
-[![Travis build status](https://img.shields.io/travis/prolificinteractive/PROJECT_NAME.svg?style=flat-square)](https://travis-ci.org/prolificinteractive/PROJECT_NAME)
-[![Cocoapods Compatible](https://img.shields.io/cocoapods/v/PROJECT_NAME.svg?style=flat-square)](https://img.shields.io/cocoapods/v/PROJECT_NAME.svg)
-[![Platform](https://img.shields.io/cocoapods/p/PROJECT_NAME.svg?style=flat-square)](http://cocoadocs.org/docsets/PROJECT_NAME)
-[![Docs](https://img.shields.io/cocoapods/metrics/doc-percent/PROJECT_NAME.svg?style=flat-square)](http://cocoadocs.org/docsets/PROJECT_NAME)
+[![Travis build status](https://img.shields.io/travis/prolificinteractive/Marker.svg?style=flat-square)](https://travis-ci.org/prolificinteractive/Marker)
+[![Cocoapods Compatible](https://img.shields.io/cocoapods/v/Marker.svg?style=flat-square)](https://img.shields.io/cocoapods/v/Marker.svg)
+[![Platform](https://img.shields.io/cocoapods/p/Marker.svg?style=flat-square)](http://cocoadocs.org/docsets/Marker)
+[![Docs](https://img.shields.io/cocoapods/metrics/doc-percent/Marker.svg?style=flat-square)](http://cocoadocs.org/docsets/Marker)
 
 ## Description
 
-Project description
+Marker anables setting Markdown formatted text on UI elements supporting `attributedText` - such as `UILabel`, `UITextField`, and `UITextView`.
 
 ## Requirements
 
-IDE, Tools etc. required for the project to run
+* iOS 8.0+
 
 ## Installation
 
-CocoaPods, Carthage, Swift Package Manager, etc
+Marker is available through [CocoaPods](http://cocoapods.org). To install it, simply add the following line to your `Podfile`:
+
+```ruby
+pod "Marker"
+```
 
 ## Usage
 
-How to use this project in 3rd party code
+Marker currently supports the following Markdown tags:
+
+* Bold (`__` or `**`)
+* Italic (`_` or `*`)
+
+### Setting Markdown Text
+
+Marker provides an extension for the following UI elements supporting `attributedText`. 
+
+Each has a `setMarkdownText:textStyle:` function - which takes in string in Markdown format as well as an object conforming to `TextStyle`.
+
+#### UILabel
+
+To set text in Markdown format for a `UILabel`:
+
+```swift
+let textStyle = MyTextStyle()
+
+myLabel.setMarkdownText("_Hello World_", textStyle: textStyle)
+```
+
+This would result in `myLabel` having the text "Hello World" displayed in italic. The specific italic font used would be specified in `MyTextStyle.`
+
+#### UITextField
+
+Setting text in Markdown format for a `UITextField` follows suit:
+
+```swift
+let textStyle = MyTextStyle()
+
+myTextField.setMarkdownText("_Hello World_", textStyle: textStyle)
+```
+
+#### UITextView
+
+Setting text in Markdown format for a `UITextView` follows suit:
+
+```swift
+let textStyle = MyTextStyle()
+
+myTextView.setMarkdownText("_Hello World_", textStyle: textStyle)
+```
 
 ## Contributing to Marker
 
```

Comment:
Could you add this sentence (Code contributions should follow the standards specified in the [Prolific Swift Style Guide](https://github.com/prolificinteractive/swift-style-guide).) about the style guide to the paragraph at line 73?